deploy/helm/node-feature-discovery-operator/templates/rbac.yaml

```
@@ -0,0 +1,369 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata: 
+  labels:
+    {{- include "node-feature-discovery-operator.labels" . | nindent 4 }}
+  name: nfd-leader-election-role
```

We prolly want to use our fullname here too(?) Like

```suggestion
  name: {{ include "node-feature-discovery-operator.fullname" . }}-leader-election-role
```